server/bin/pbench-dispatch.sh

```diff
@@ -104,17 +104,35 @@ if [[ ${sts} != 0 ]]; then
     exit 3
 fi
 
-# The following directory holds tar balls that are quarantined because
-# of operational errors on the server. They should be retried after
-# the problem is fixed: basically, move them back into the reception
-# area for 002 agents and wait.
-errors=${qdir}/errors-${version}
-mkdir -p ${errors}
-sts=${?}
-if [[ ${sts} != 0 ]]; then
-    echo "Failed: \"mkdir -p ${errors}\", status ${sts}" >> ${errlog}
-    exit 3
-fi
```

Comment:
This removal isn't mentioned in the PR description nor proposed Git log.  (Not that I'm objecting...I just want to make sure that it is supposed to be part of this PR and not stay in #3299.)

Reply:
Yes, this was supposed to be a separate change.  Fixed the commit comment as well as the PR description.